Mathlib/MeasureTheory/Integral/SetIntegral.lean

```diff
@@ -1298,6 +1298,12 @@ theorem integral_re {f : X → 𝕜} (hf : Integrable f μ) :
     ∫ x, RCLike.re (f x) ∂μ = RCLike.re (∫ x, f x ∂μ) :=
   (@RCLike.reCLM 𝕜 _).integral_comp_comm hf
 
+theorem Complex.integral_re {f : X → ℂ} (hf : Integrable f μ) :
+    ∫ x, (f x).re ∂μ = (∫x, f x ∂μ).re := _root_.integral_re hf
+
```

Comment:
Can you say what this is needed for? I am wondering why it isn't covered by `integral_re` above, since `ℂ` is of course an example of an `RCLike` field. If it is needed, then we should presumably have the `im` version too (and likewise for the next lemma). Note also spacing `∫ x` not `∫x`.

(Incidentally, I admit that I'm a bit puzzled why the root-level lemma `integral_re` is placed in this file `SetIntegral`, since it doesn't actually involve set integrals! But that is a question for another PR.)